File: Dev/51900718/src/main/java/tdtu/advanced/java/thinh68/models/Lichhen.cs
namespace ComNha.Models
{
    public class Lichhen
    {
        public string MaLichHen { get; set; } = null!;
        public string? NhuCau { get; set; }
        public int SoLuongKhach { get; set; }
        public TimeSpan ThoiGian { get; set; }
        public DateTime NgayHen { get; set; }
        public int? TinhTrang { get; set; }
        public int MaKhachHang { get; set; }
        public string? MaNhanVien { get; set; }
        public string? MaBanAn { get; set; }
    }
}

File: Dev/Models/Models/LichHen.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;


@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name="lichhen")
public  class LichHen {
    @Id
    @GeneratedValue
    private Long MaLichHen;
    private String NhuCau;
    private int SoLuongKhach;
    public TimeSpan ThoiGian;
    private DateTime NgayHen;
    private int TinhTrang;
    private int MaKhachHang;
    private String MaNhanVien;
    private String MaBanAn;
}